packages/catalog-process/src/services/catalogService.ts

```diff
@@ -481,103 +735,20 @@ export function catalogServiceBuilder(
 
     async createEService(
       seed: catalogApi.EServiceSeed,
-      { authData, correlationId, logger }: WithLogger<AppContext>
+      ctx: WithLogger<AppContext>,
+      eServiceTemplateReferences?: EServiceTemplateReferences
```

Comment:
This seems to be never passed? Also, there's another method to create an e-service from a template instance, this one should probably never have template references

```suggestion
```